Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/es/Installation-for-Linux.po:
```diff
@@ -101,34 +101,29 @@ msgstr "*Solo Ubuntu* - Habilita el repositorio \"universe\" de Ubuntu (puedes u
-#| msgid "Run the setup script and install Jamulus: `sudo ./setup_repo.sh && sudo apt install jamulus`"
 msgid "Run the setup script and install Jamulus:\\\\"
-msgstr "Ejecuta el script de instalación e instala Jamulus: `sudo ./setup_repo.sh && sudo apt install jamulus`"
+msgstr "Ejecuta el script de instalación e instala Jamulus:\\\\"
```
@ignotus666 this is actually new.

ignotus666 replied:
Yeah, I just realised. A few other strings had turned into fuzzies (only on Weblate - they were ok here) but it's all back to 100% now I think.

ann0see reviewed (commented):
I fixed a lot of stuff on the german site. So this becomes big.

ann0see commented on _translator-files/po/de/Contribution.po:
```diff
@@ -43,38 +43,32 @@ msgstr "Beitragen"
+#, no-wrap
 msgid "If you think some documentation or information is missing or can be improved, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">post about it on this forum</a> so it can be discussed first.\n"
-msgstr "Wenn du der Meinung bist, dass eine Dokumentation oder eine Information fehlt oder verbessert werden kann, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">poste deine Meinung in diesem Forum</a>, damit es zuerst diskutiert werden kann."
+msgstr "Wenn du der Meinung bist, dass eine Dokumentation oder eine Information fehlt oder verbessert werden kann, <a href=\"https://github.com/jamulussoftware/jamulus/discussions\">poste deine Meinung in diesem Forum</a>, damit es zuerst diskutiert werden kann.\n"
```
Needs rewording. Eine Dokumentation oder eine Information fehlt is not valid German.

ann0see commented on _translator-files/po/de/Directories.po:
```diff
@@ -44,45 +44,45 @@ msgstr "Verzeichnisse"
 #, no-wrap
 msgid "To view Servers listed by a Custom Directory, users must enter the address of that Directory in their Client's Settings > Advanced Setup > Custom Directories field. Multiple addresses can be added in this way if needed. Custom Directories will then appear in the Directory drop-down list on their Connect window. Custom Directories otherwise work for Clients in the same way as Public Directo ...
-msgstr ""
+msgstr "Um die von einem benutzerdefinierten Verzeichnis aufgelisteten Server zu sehen, müssen Benutzer die Adresse dieses Verzeichnisses in das Feld Einstellungen > Erweiterte Einstellungen> Benutzerdefinierte Verzeichnisse ihres Clients eingeben. Bei Bedarf können mehrere Adressen auf diese Weise hinzugefügt werden. Benutzerdefinierte Verzeichnisse erscheinen dann in der Dropdownliste Verzeichn ...
```
```suggestion
msgstr "Um die von einem benutzerdefinierten Verzeichnis aufgelisteten Server zu sehen, müssen Benutzer die Adresse dieses Verzeichnisses in das Feld Einstellungen > Erweiterte Einstellungen > Benutzerdefinierte Verzeichnisse ihres Clients eingeben. Bei Bedarf können mehrere Adressen auf diese Weise hinzugefügt werden. Benutzerdefinierte Verzeichnisse erscheinen dann in der Dropdownliste Verzeichnis im Verbinden Fenster. Benutzerdefinierte Verzeichnisse funktionieren für Clients auf dieselbe Weise wie öffentliche Verzeichnisse und zeigen eine Liste der bei ihnen registrierten Server an.\n"
```

ann0see commented on _translator-files/po/de/FAQ.po:
```diff
@@ -89,17 +89,14 @@ msgstr "**Häufige Probleme und deren Lösungen bei der Verwendung von Jamulus f
+#, no-wrap
 msgid "Is there a metronome, synchronization, or some other way of keeping in time?"
-msgstr "### Hat Jamulus ein Metronom, eine Synchronisation oder eine andere Möglichkeit, den Takt zu halten?"
+msgstr "Hat Jamulus ein Metronom, eine Synchronisation oder eine andere Möglichkeit, den Takt zu halten?"
```
```suggestion
msgstr "Hat Jamulus ein Metronom, Synchronisation oder eine andere Möglichkeit, den Takt zu halten?"
```

ann0see commented on _translator-files/po/de/FAQ.po:
```diff
@@ -108,31 +105,25 @@ msgstr "Beachte dass sich alle Teilnehmer an die [Goldene Regel](/wiki/Client-Tr
-#| msgid "In general, if somebody lists a Server on one of the Public Directories provided by Jamulus by default, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms built in. However, some Servers may state their policies in the welcome message you will see in the chat window."
 msgid "If somebody lists a Server on one of the Directories built into Jamulus, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. Whilst some Servers may state their policies in the welcome message you will see in the chat window, Jamulus itself does nothing to enforce these."
-msgstr "Wenn jemand einen Server in eines der öffentlichen Verzeichnisse einträgt, die Jamulus standardmäßig bereitstellt, akzeptiert er damit, dass jeder auf ihm spielen kann. Jamulus unterstützt keinen Passwortschutz oder andere Authentifizierungsmechanismen. Einige Server können jedoch in der Willkommensnachricht, die du im Chat-Fenster sehen kannst, Richtlinien angeben."
+msgstr "Wenn jemand einen Server in eines der standardmäßig in Jamulus integrierten Verzeichnisse einträgt, akzeptiert er damit, dass jeder auf ihm spielen kann. Jamulus unterstützt keinen Passwortschutz oder andere Authentifizierungsmechanismen. Obwohl einige Server in ihrer Willkommensnachricht, die du im Chat-Fenster siehst, Richtlinien angeben, wird Jamulus nicht versuchen sie durchzusetzen."
```
```suggestion
msgstr "Wenn jemand einen Server in eines der standardmäßig in Jamulus integrierten Verzeichnisse einträgt, akzeptiert er damit, dass jeder auf ihm spielen kann. Jamulus unterstützt keinen Passwortschutz oder andere Authentifizierungsmechanismen. Obwohl einige Server in ihrer Willkommensnachricht - die du im Chat-Fenster siehst - Richtlinien angeben, wird Jamulus nicht versuchen sie durchzusetzen."
```

ann0see commented on _translator-files/po/de/FAQ.po:
```diff
@@ -238,10 +220,8 @@ msgstr "Normalerweise verhindern NAT-Firewalls (Network Address Translation) ein
 #: ../wiki/en/FAQ.md:73
-#, fuzzy
-#| msgid "However, in Unregistered mode, Clients have to **initiate** connections into the Server’s network. NAT firewalls prevent this, so you need to tell them to allow incoming connection requests on the Jamulus port using port forwarding."
 msgid "However, to reach an Unregistered Server, each Client has to **initiate** a connection into the Server’s network. A NAT firewall prevents this, so you need to configure the firewall to allow incoming connection requests on the Jamulus port using port forwarding."
```
NAT is not a Firewall - this should be changed.

ann0see commented on _translator-files/po/de/Include-Client-Commands.po:
```diff
@@ -1,62 +1,46 @@
-#| msgid "`--ctrlmidich`  MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)\n"
 msgid "`--ctrlmidich` MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)"
-msgstr "`--ctrlmidich`  MIDI-Controller-Kanal, der genutzt werden soll, Offset der Kontrollnummer und fortlaufende CC-Nummern (Kanäle) und Mute Myself CC-Nummer. Format: `kanal[;f*][;p*][;s*][;m*][;o]` Siehe [Tipps & Tricks](Tips-Tricks-More#verwendung-von-ctrlmidich-für-midi-controller) \n"
+msgstr "`--ctrlmidich` MIDI-Controller-Kanal, der genutzt werden soll, Offset der Kontrollnummer und fortlaufende CC-Nummern (Kanäle) und Mute Myself CC-Nummer. Format: `kanal[;f*][;p*][;s*][;m*][;o]` Siehe [Tipps & Tricks](Tips-Tricks-More#verwendung-von-ctrlmidich-für-midi-controller)"
```
```suggestion
msgstr "`--ctrlmidich` MIDI-Controller-Kanal, der genutzt werden soll, Offset der Kontrollnummer und fortlaufende CC-Nummern (Kanäle) und Stummschalten CC-Nummer. Format: `kanal[;f*][;p*][;s*][;m*][;o]` Siehe [Tipps & Tricks](Tips-Tricks-More#verwendung-von-ctrlmidich-für-midi-controller)"
```

ann0see commented on _translator-files/po/de/Tips-Tricks-More.po:
```diff
@@ -96,37 +94,32 @@ msgstr "Jamulus-Benutzer [Chris Rimple](https://sourceforge.net/u/chrisrimple/pr
-#| msgid "You can have a \"private\", undisturbed session with other people on any Server, including publically listed Servers, by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information)."
 msgid "You can have an undisturbed session with other people on any Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server."
-msgstr "Du kannst eine „private“, ungestörte Sitzung mit anderen Personen auf einem beliebigen Server, einschließlich öffentlich gelisteten Servern, abhalten, indem sich alle Teilnehmer auf SOLO schalten. Du wirst dann nicht in der Lage sein, andere Personen zu hören, wenn diese den selben Server betreten (beachte, dass dies andere nicht daran hindert, dich zu hören, die Chat-Funktion zu benutzen ...
+msgstr "Du kannst eine ungestörte Sitzung mit anderen Personen auf einem beliebigen Server abhalten, indem sich alle Teilnehmer auf SOLO schalten. Du wirst dann nicht in der Lage sein, andere Personen zu hören, wenn diese den selben Server betreten."
```
```suggestion
msgstr "Du kannst eine ungestörte Sitzung mit anderen Personen auf einem beliebigen Server abhalten, indem sich alle Teilnehmer auf SOLO schalten. Du wirst andere Personen nicht hören, wenn diese den selben Server betreten."
```

ann0see commented on _translator-files/po/de/Tips-Tricks-More.po:
```diff
@@ -96,37 +94,32 @@ msgstr "Jamulus-Benutzer [Chris Rimple](https://sourceforge.net/u/chrisrimple/pr
 #, no-wrap
 msgid "**NOTE:** This does not stop people from hearing you, using the chat function, or seeing your profile information.\n"
-msgstr ""
+msgstr "**HINWEIS:** Das hindert niemanden daran, dich zu hören, die Chat-Funktion zu nutzen oder deine Profilinformationen zu sehen.\n"
```
```suggestion
msgstr "**HINWEIS:** Das hindert niemanden daran dich zu hören, die Chat-Funktion zu nutzen oder deine Profilinformationen zu sehen.\n"
```

ann0see commented on _translator-files/po/de/Tips-Tricks-More.po:
```diff
@@ -323,31 +285,26 @@ msgstr "*Tipp*: Wenn du die MIDI-Steuerung in Jamulus aktivierst, wird dem Namen
 #: ../wiki/en/Tips-Tricks-More.md:106
-#, fuzzy
-#| msgid "You can run a public Server long enough for your band to connect, then go private by simply unchecking the 'Make my Server Public' box in the Server GUI. Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)"
 msgid "You can run as a Registered Server long enough for your band to connect, then go \"private\" (Unregistered) by setting the Directory to \"none\" in the Server GUI. Your band mates will still be connected to the Server until they disconnect. (Thanks to [David Savinkoff](https://github.com/DavidSavinkoff) for this tip!)"
```
We should get away from saying "band" here.

pljones replied:
Bug against the English version?  I'm not a fan of it, either.  "Your clients" would be correct but sounds overly formal...

ann0see replied:
Yes

ann0see replied:
See #951

ann0see commented on _translator-files/po/de/Unregistered-Servers.po:
```diff
@@ -33,34 +33,34 @@ msgstr "wiki"
 #: ../wiki/en/Unregistered-Servers.md:13
 msgid "It is highly recommended to test your Server by registering it on one of the built-in Directories **first**. This will help you tackle general problems before you try unregistered mode."
-msgstr ""
+msgstr "Es ist dringend empfohlen, den Server zu testen, indem du ihn **zuerst** an einem der integrierten Verzeichnisse registrierst. So kannst du allgemeine Probleme beheben, bevor du den unregistrierten Modus nutzt."
```
```suggestion
msgstr "Es ist dringend empfohlen den Server zu testen, indem du ihn **zuerst** an einem der integrierten Verzeichnisse registrierst. So kannst du allgemeine Probleme beheben, bevor du den unregistrierten Modus nutzt."
```

ann0see commented on _translator-files/po/de/Unregistered-Servers.po:
```diff
@@ -71,78 +71,79 @@ msgstr "Portweiterleitung"
 #: ../wiki/en/Unregistered-Servers.md:28
 msgid "To allow others to connect to your Server from the internet, give them your public internet IP address. You can [learn your current address using Google](https://www.google.com/search?q=what+is+my+ip).  You should connect yourself using the local network address of the computer the Server is running on. If you are running a Client on the same computer as your Server, the address is `localh ...
-msgstr ""
+msgstr "Damit andere sich mit deinem Server über das Internet verbinden können, gib ihnen deine öffentliche Internet-IP-Adresse. Du kannst [deine aktuelle Adresse mit Google ermitteln] (https://www.google.de/search?q=was+ist+meine+ip).  Du selbst solltest dich über die lokale Netzwerkadresse des Computers verbinden, auf dem der Server läuft. Wenn du einen Client auf demselben Computer wie den Ser ...
```
Public internet IP address is redundant.

ann0see commented on _translator-files/po/fr/QOS-Windows.po:
```diff
@@ -60,12 +61,12 @@ msgstr "Jamulus utilise la qualité de service (QoS) pour atténuer les délais
 #: ../wiki/en/QOS-Windows.md:15
 msgid "If you want to explore the effect of non-default settings, see [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) pages 16-18.  Jamulus sets the DS Field byte to 128 (or 0x80) to select DSCP/CS4 in an IPv4 or IPv6 packet header. Other byte values can be set using the -Q option, e.g. -Q [0..255].  However, most people will have no need to do this."
-msgstr "Si vous souhaitez explorer l'effet des paramètres par défaut, consultez [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) pages 16-18. Jamulus définit l'octet DS Field à 128 (ou 0x80) pour sélectionner DSCP/CS4 dans un en-tête de paquet IPv4 ou IPv6. D'autres valeurs d'octets peuvent être définies en utilisant l'option -Q, par exemple -Q [0..255]. Cependant, la plupart des gens n'a ...
+msgstr "Si vous souhaitez explorer l'effet de paramètres qui ne sont pas ceux par défaut, consultez [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) pages 16-18. Jamulus définit l'octet DS Field à 128 (ou 0x80) pour sélectionner DSCP/CS4 dans une en-tête de paquet IPv4 ou IPv6. D'autres valeurs d'octets peuvent être définies en utilisant l'option -Q, par exemple -Q [0..255].  Cependant, l ...
```
These files might need to be checked.

ann0see reviewed (changes requested):
Please review.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/include-client-commands/
Translation: Jamulus/Include-Client-Commands

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Directories
Translate-URL: https://hosted.weblate.org/projects/jamulus/directories/

ann0see commented:
@henkdegroot @trebmuh @jujudusud @ignotus666 please Review This.

jujudusud commented:
Ok for French part.

Le mar. 1 août 2023, 23:48, ann0see ***@***.***> a écrit :

> @henkdegroot <https://github.com/henkdegroot> @trebmuh
> <https://github.com/trebmuh> @jujudusud <https://github.com/jujudusud>
> @ignotus666 <https://github.com/ignotus666> please Review This.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jamulussoftware/jamuluswebsite/pull/948#issuecomment-1661145218>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADBIPYGL3522OWCHDIZMNHDXTF2SDANCNFSM6AAAAAA26ZHTQQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

trebmuh commented:
:+1:

ann0see commented:
I've now locked the website on Weblate to avoid any conflicts. I'll unlock it once some PRs are merged.

trebmuh commented:
> I've now locked the website on Weblate to avoid any conflicts. I'll unlock it once some PRs are merged.

OK, I'm putting my translation reviews on pause then.
I still have a few manual page French translation to review, and the app translation too.
